### ISSUE TITLE
Smooth Scroll does not work well in all scrollable sections.

### ISSUE THREAD

**Issue by s1davide** [Caldis/Mos#622]
**Describe the bug | 问题描述**
Smooth scrolling works well in most scrollable sections of an app but in some it seems to ignore it, such as the "System Settings" example.

**To Reproduce | 如何重现**
Open System Settings App and scroll the right section.

**Expected behavior | 期望结果**
Smooth Scroll in all scrollable sections.

**Screenshots | 相关截图**
![scroll-explanation](https://github.com/Caldis/Mos/assets/60453099/483aaba4-e5c8-4f92-b069-0f39f712d1b6)

**System Info | 系统信息:**
 - Mouse: Logitech MX Master 3 and MX Master 3s
 - OS: Mac OSX Sonoma
 - Browser safari, chrome
 - Version 17.1
 -  Application System Settings, Other scroll blocks in apps

**Comment by vladcudoidem** [comment]
For me, scrolling works well in System Settings, but does not work in the Applications Popup that comes up when you click on the "Applications" icon in the dock.

<img width="978" alt="Screenshot 2024-05-12 at 15 33 49" src="https://github.com/Caldis/Mos/assets/52567079/29493e4b-dc25-4e4b-aa72-9ca7b549525e">

**Comment by s1davide** [comment]
> For me, scrolling works well in System Settings, but does not work in the Applications Popup that comes up when you click on the "Applications" icon in the dock.
> 
> <img alt="Screenshot 2024-05-12 at 15 33 49" width="978" src="https://private-user-images.githubusercontent.com/52567079/329853486-29493e4b-dc25-4e4b-aa72-9ca7b549525e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MzkyMzksIm5iZiI6MTcxNTUzODkzOSwicGF0aCI6Ii81MjU2NzA3OS8zMjk4NTM0ODYtMjk0OTNlNGItZGMyNS00ZTRiLWFhNzItOWNhN2I1NDk1MjVlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEyVDE4MzUzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA0YThjZmY0YTljOGEwMzZkOWRhMzM2MWFkZDI4NGE0ZGU1NGRkMzE3MmZiMDQ3OGIwMDU4OGM0ZGY5ZDkxNjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KrA4oXKiad6C800JOIjAlBLnEkBenoMfzhh5PDojiq8">
Are you sure that it works well in the sections on the right?, like in "System Settings"?, as well as the image that I put, on the left side smooth scrolling works, the right side will look laggy because MOS cannot apply smooth scrolling to that area

**Comment by vladcudoidem** [comment]
Yes, for me it works perfectly in that area as well. What version of MOS do you have?
